src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3076,12 +3167,12 @@ namespace SimAirServingZones {
         // FLOW:
 
         BypassOAController = false; // simulate OA water coil controllers
-        AirLoopPass = AirLoopControlInfo(AirLoopNum).AirLoopPass;
+        if (AirLoopNum > 0) AirLoopPass = AirLoopControlInfo(AirLoopNum).AirLoopPass;
         IsUpToDateFlag = false;
-        PrimaryAirSystem(AirLoopNum).ControlConverged = false;
+        if (AirLoopNum > 0) PrimaryAirSystem(AirLoopNum).ControlConverged = false;
 
         AllowWarmRestartFlag = true;
-        AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag = true;
+        if (AirLoopNum > 0) AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag = true;
```

Comment:
Extra logic checks are a waste of computation.  Could this AirLoopNum be checked once and then a series of things be done only for that condition?

Reply:
@Myoldmopar Done. Thanks.